code/modules/mob/living/carbon/carbon.dm

```diff
@@ -572,7 +572,7 @@
 	set_health(round(maxHealth - getOxyLoss() - getToxLoss() - total_burn - total_brute, DAMAGE_PRECISION))
 	update_stat()
 	update_stamina()
-	if(((maxHealth - total_burn) < HEALTH_THRESHOLD_DEAD*2) && stat == DEAD )
+	if(((maxHealth - total_burn) < husk_threshold) && stat == DEAD )
```

Comment:
This value should be determined by bodypart max health, not by an arbitrary number

For example, human chests have an hp of 200, so their husk threshold is -200
Alien chests are 500, so logically it'd be -500

So something like
```dm
var/husk_thresold = get_bodypart(BODY_ZONE_CHEST).max_damage * -1
```